SPECS/xen.spec

```diff
@@ -1100,6 +1100,9 @@ fi
 %{?_cov_results_package}
 
 %changelog
+* Tue Apr 29 2025 Yann Dirson <yann.dirson@vates.tech> - 4.17.5-9.2
+- Rebuild against newer ncurses to pull abi5 (compat) libs
```

Comment:
Non-blocking suggestion: as "newer" is a bit confusing here where we're actually using somewhat older/compat libs, I'd rather mention which EVR of nurses we build against.

I haven't reviewed the other PRs yet, but if the changelog the same there, then this likely applies to them to (I won't add more comments about it).

Reply:
Fixed for all three PRs